EVENT.md

```diff
@@ -824,6 +825,41 @@ debug reports without cross-site leak.
 TODO: Consider supporting debug reports for attribution registrations inside a
 fenced frame tree.
 
+### Optional: header errors debugging reports
```

Comment:
nit: errors->error

Reply:
Already changed to header-error debugging reports.